frontend/src/locales/en.json

```diff
@@ -275,10 +301,19 @@
         "prover": "Prover",
         "verifier": "Verifier"
       }
-   }
+    }
   },
   "showRawData": "Show raw data",
   "error": {
     "axios": "Request failed with status {statusText}"
+  },
+  "common":{
+    "tables":{
+      "headers":{
+        "last_update_date":"Last Updated",
```

Comment:
lastUpdateDate and createdBy. Let's keep consistent and make sure the VUE files use the correct name... Are we using these? If not, then just remove them from both resource files.